portfolio/src/main/webapp/second.html

```diff
@@ -0,0 +1,25 @@
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <title>About me</title>
+    <link rel="stylesheet" href="style.css">
+    <div class="topnav">
+        <a class="active" href="index.html">Home</a>
+        <a href="#second">About me</a>
+        <a href="projects.html">Projects</a>
+        <a href="skills.html">Skills</a>
+        <a href="contact.html">Contact</a>
+    </div>
+</head>
+<header id="second">
```

Comment:
We usually like to have descriptive IDs that will tell us what specifically the object is. So perhaps giving this an ID such as "about-me-header" or something similar would be better.